ts/components/InfoAltScreenComponent/InfoAltScreenComponent.tsx

```diff
@@ -0,0 +1,73 @@
+import { NavigationEvents } from "@react-navigation/compat";
+import { View as NBView } from "native-base";
+import * as React from "react";
+import { StyleSheet, Text, View } from "react-native";
+import themeVariables from "../../theme/variables";
+import { setAccessibilityFocus } from "../../utils/accessibility";
+import { IOPictogramType, Pictogram } from "../core/pictograms";
+import { Body } from "../core/typography/Body";
+import { H2 } from "../core/typography/H2";
+
+type Props = {
+  image: IOPictogramType;
+  title: string;
+  body?: string | React.ReactNode;
+};
+
+const styles = StyleSheet.create({
+  main: {
+    padding: themeVariables.contentPadding,
+    flex: 1,
+    alignItems: "center",
+    justifyContent: "center"
+  },
+  textAlignCenter: {
+    textAlign: "center"
+  }
+});
+
+const renderNode = (body: string | React.ReactNode) => {
+  if (typeof body === "string") {
+    return (
+      <Body testID="InfoAltScreenBody" style={styles.textAlignCenter}>
+        {body}
+      </Body>
+    );
+  }
+
+  return body;
+};
+
+/**
+ * A base screen that displays one illustration (through the Pictogram component),
+ * text, and one bottom button. It's an alternative to the InfoScreenComponent
+ * that accepts a generic raster image.
+ *
+ * @param props
+ * @constructor
+ */
+export const InfoAltScreenComponent = ({ image, title, body }: Props) => {
+  const elementRef = React.createRef<Text>();
+
+  return (
+    <View style={styles.main} testID="InfoAltScreenComponent">
+      <NavigationEvents onWillFocus={() => setAccessibilityFocus(elementRef)} />
+      <Pictogram name={image} />
+      <NBView spacer={true} large={true} />
+      <H2
+        testID="infoScreenTitle"
+        accessible
+        ref={elementRef}
+        style={styles.textAlignCenter}
+      >
+        {title}
+      </H2>
+      {body && (
+        <React.Fragment>
+          <NBView spacer={true} />
+          {renderNode(body)}
+        </React.Fragment>
```

Comment:
```suggestion
        <>
          <NBView spacer={true} />
          {renderNode(body)}
        </>
```
There's no need to declare directly a Fragment 😄